lib/twilio/audiohelper.ts

```diff
@@ -342,6 +348,35 @@ class AudioHelper extends EventEmitter {
     });
   }
 
+  /**
+   * Adds an {@link AudioProcessor}. If an {@link AudioProcessor} exists in the
+   * {@link AudioHelper}, the {@link AudioHelper} routes the input audio stream
+   * to the {@link AudioProcessor}.
+   *
+   * The {@link AudioHelper} guarantees that this audio stream is always
+   * the active input audio stream and will automatically
+   * update whenever the user's preference changes.
+   *
+   * Only one {@link AudioProcessor} can be added at this time.
+   * @param processor
+   */
+  addProcessor(processor: AudioProcessor): void {
+    if (this._processor) {
+      throw new NotSupportedError('Adding multiple AudioProcessors is not supported at this time.');
+    }
+
+    if (!processor) {
```

Comment:
This change will make sure a value of `null` will not trigger this error:
```suggestion
    if (typeof processor === 'undefined') {
```

Reply:
We don't want the user to set this to null using this method. I'll check for object and null instead.